package-lock.json

```diff
@@ -53,6 +52,7 @@
         "terser": "5.19.2",
         "tslib": "2.6.1",
         "typescript": "5.1.6",
+        "urlpattern-polyfill": "9.0.0",
```

Comment:
Is this change deliberate?

Reply:
The `package-lock.json` wasn't updated before.